py3status/modules/bluetooth.py

```diff
@@ -3,84 +3,109 @@
 Display bluetooth status.
 
 Configuration parameters:
-    cache_timeout: how often we refresh this module in seconds (default 10)
-    device_separator: the separator char between devices (only if more than one
-        device) (default '|')
-    format: format when there is a connected device (default '{name}')
-    format_no_conn: format when there is no connected device (default 'OFF')
-    format_no_conn_prefix: prefix when there is no connected device
-        (default 'BT: ')
-    format_prefix: prefix when there is a connected device (default 'BT: ')
+    always_collapse: always collapse to hide format_bt (default False)
+    button_toggle: mouse button to toggle visibility (default 1)
+    cache_timeout: refresh interval for this module (default 10)
+    format: display format for this module (default 'BT: {format_bt}')
+    format_bt: display format for bluetooth (default '{name}')
+    string_separator: show separator only if more than one (default '|')
 
 Format placeholders:
-    {name} device name
-    {mac} device MAC address
+    {format_bt} display format for bluetooth
+
+format_bt placeholders:
+    {name} bluetooth device name
+    {mac} bluetooth device address
 
 Color options:
-    color_bad: Connection on
-    color_good: Connection off
+    color_bad: No connection
+    color_good: Active connection
 
 Requires:
-    hcitool:
+    hcitool: configure Bluetooth connections (may be in bluez*)
 
 @author jmdana <https://github.com/jmdana>
 @license GPLv3 <http://www.gnu.org/licenses/gpl-3.0.txt>
 """
 
 import re
 BTMAC_RE = re.compile(r'[0-9A-F:]{17}')
+string_unavailable = "bluetooth: hcitool isn't installed"
 
 
 class Py3status:
     """
     """
     # available configuration parameters
+    always_collapse = False
+    button_toggle = 1
     cache_timeout = 10
-    device_separator = '|'
-    format = '{name}'
-    format_no_conn = 'OFF'
-    format_no_conn_prefix = 'BT: '
-    format_prefix = 'BT: '
+    format = 'BT: {format_bt}'
+    format_bt = '{name}'
+    string_separator = '|'
+
+    class Meta:
+        deprecated = {
+            'rename': [
+                {
+                    'param': 'device_separator',
+                    'new': 'string_separator',
+                    'msg': 'obsolete parameter use `string_separator`',
+                },
+            ],
+        }
+
+    def post_config_hook(self):
+        self.format_bt_info = self.format_bt
+        self.format_info = self.format
```

Comment:
that can be done much more cleanly - but anyhow I think it should die - but maybe I haven't looked at the code enough - I'll do that at some point but need to find my bt dongle